### DISCUSSION TITLE
New Fresh Jr. Developer Doesn't know where to go.

### DISCUSSION THREAD

**Discussion by MyBaBB** [community/community#68846] [GitHub Education]
### Hi everyone,

This is an old post, I have advanced Significantly since then,
I am on React js. now and It is a whole new ballgame. 
I am still looking for work doing something, having no luck at all.
I thought I had a Movie gig,
It was a fun project,  They cut it from the script.
It was fun.  I will make it public now.
its unfinished and I didn't break it
down yet but you see where I was going.

**Comment by TionKior** [comment]
I think you can try to find a job after you have some skills, because work can speed up the exercise of your skills, much faster than learning Xi on your own

**Reply by MyBaBB** [reply]
Thank you, That is what  I have discovered.   I have been building things for about 6 months now,   There should be somewhere that would take me in as an intern.  I just made my own Internship here on Github working for Myself.  I lost a lot of green squares when I Deleted some my repos.   Bummer.
I just search on Craigslist and send out emails.  Not a lot of luck there.  Everyone is Wordpress or Drupal and I did not want to get caught up in that.  I am going for the good stuff.  I doesn't appear that I built very many things but I have built the same things over and over and over until I got it.

**Comment by samus-aran** [comment]
Hi @MyBaBB 

Welcome to the community. 👋  Building friendships here (and any developer community) requires a combination of genuine engagement, active participation, and effective communication. I'm not an expert but I would suggest the following to help make new friendships and connect with your peers:

- **Introduce yourself in the appropriate area of the community** (here isn't quite right unless you are a student yourself)
- **Engage in discussions relevant to your interests with genuine participation** (comments that contribute to the topic of the discussion)
- **Help and support others (Questions are great but try and seek if you can help others too)** - Providing support not only helps others but also showcases your expertise and willingness to contribute.
- **Maintain a positive and respectful tone in your interactions.** Treat others with kindness and professionalism, even in disagreements. Also post with substance - commenting for the sake of "bumping" up a discussion or having a conversation with one's own self is discouraged. 😉 

Remember, building friendships takes time, so be patient and consistent in your efforts to connect with others.

Lastly, "to go from here" would be to search the discussion board for conversations on topics you're interested in, I'm sure there are many other community members whom have already asked how to progress with their learning. Perhaps the Programming Help category would be a good place to start.

Good Luck 👍

**Comment by MyBaBB** [comment]
Oh boy, thanks.   I really needed guidance when I first started.  I didn't have a clue.  I didn't even know what html was. I just kept my nose to the grind stone.  I can make websites now. I really needed the community help but I didn't know where they where. Thank you so much for saying something to me. It might be discouraged but at least someone said something.   There should be an easier way for some of someone who knows absolutely  nothing.
Yes I am still a student and probably always will be.
I spend countless night trying to figure out what's next.
I can build 💪 react now. And still feel like I know nothing.

Sent from my MetroPCS 5G Android device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
